docs/docsite/rst/network/user_guide/network_working_with_command_output.rst

```diff
@@ -63,3 +66,34 @@ The ``wait_for`` argument must always start with result and then the
 command index in ``[]``, where ``0`` is the first command in the commands list,
 ``1`` is the second command, ``2`` is the third and so on.
 
+
+Handling prompts in network modules
+===================================
+
+Network devices may require that you answer a prompt before performing a change on the device. Individual network modules such as :ref:`ios_command <ios_command_module>` and :ref:`nxos_command <nxos_command_module>` can handle this with a ``prompt`` parameter.
+
+You can also use the :ref:`cli_command <cli_command_module>` to handle multiple prompts.
+
+.. code-block:: yaml
+
+  ---
+   - name: reboot ios device
+     cli_command:
+       command: reload
+       prompt:
+         - Save?
+         - confirm
+       answer:
+         - y
+         - y
+
+This task answers yes to both prompts. You must list the prompt and the answers in the same order (that is, prompt[0] is answered by answer[0]).
```

Comment:
Since `check_all: True` is missing for all the matching prompts it will just send the first answer in the list that is `y`.
If the use case requires a different answer for each prompt in that case the `check_all` option should be set to true and the sequence for prompt should match with the corresponding entry in answer.
For example:
```
- name: multiple prompt, multiple answer (mandatory check for all prompts)
  cli_command:
    command: "copy sftp sftp://user@host//user/test.img"
    check_all: True
    prompt:
      - "Confirm download operation"
      - "Password"
      - "Do you want to change that to the standby image"
    answer:
      - 'y'
      - <password>
      - 'y'
```

Reply:
thanks, added this example to the doc.